cpu/lm4f120/Makefile.include

```diff
@@ -1,8 +1,10 @@
 export CPU_ARCH = cortex-m4f
 
-include $(RIOTMAKE)/arch/cortexm.inc.mk
-
 # use common periph functions
 USEMODULE += periph_common
+USEMODULE += pm_cortexm_common_set_lowest
```

Comment:
IMO we can generalize this for all cortexm boards:
if not (feature_provided(periph/pm)) -> use ...._common_set_lowest
of make it dependend on whether pm_layered is used.

Reply:
Sounds good. As mentioned above we have to check if all boards based on cortexm_common AND stm32_common can still use the pm implementation of cortexm_common!

Reply:
well, ```pm_reboot()``` will definitely be the same for all cortexm. "pm_off()" will be cpu dependent. "pm_set_lowest()" will be the same for all cortexm not implementing periph/pm.

Reply:
where would we implemented such a check to select all cortexm_common fallback if the feature is not provided by the cpu itself?